presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotClusterInfoFetcher.java

```diff
@@ -359,10 +395,8 @@ public RoutingTables(@JsonProperty("routingTableSnapshot") List<RoutingTableSnap
             else {
                 List<Map<String, List<String>>> routingTableEntriesList = snapshot.getRoutingTableEntries();
                 if (routingTableEntriesList.isEmpty()) {
-                    throw new PinotException(
-                            PINOT_UNEXPECTED_RESPONSE,
-                            Optional.empty(),
-                            String.format("Empty routingTableEntries for %s. RoutingTable: %s", tableName, responseBody));
+                    throw new PinotException(PINOT_UNEXPECTED_RESPONSE, Optional.empty(),
+                        String.format("Empty routingTableEntries for %s. RoutingTable: %s", tableName, responseBody));
```

Comment:
either put all parameters in one line, or one in each line

Reply:
done